llvm/lib/TargetParser/AArch64TargetParser.cpp

```diff
@@ -114,6 +121,10 @@ const AArch64::ArchInfo *AArch64::parseArch(StringRef Arch) {
 }
 
 std::optional<AArch64::ExtensionInfo> AArch64::parseArchExtension(StringRef ArchExt) {
+  // Resolve aliases first.
+  ArchExt = resolveExtAlias(ArchExt);
```

Comment:
Seems resolving aliases here expands "rdma" alias beyond FMV.

Reply:
I will separate this change to another PR and leave this review just for the adjustment of feature priorities.